src/libstd/sync/mpsc/mod.rs

```diff
@@ -949,16 +1145,17 @@ impl<T> Receiver<T> {
     ///
     /// This function will always block the current thread if there is no data
     /// available and it's possible for more data to be sent. Once a message is
-    /// sent to the corresponding [`Sender`], then this receiver will wake up and
-    /// return that message.
+    /// sent to the corresponding [`Sender`] (or [`SyncSender`]), then this
```

Comment:
Hmm, how did Travis not pick it up last time. :confused: Or the tests I ran, for that matter.